exploring_UV_extinction_curves_pt1/downloading_UV_data_pt1.ipynb

```diff
@@ -0,0 +1,1433 @@
+{
```

Comment:
<p>I just wonder, stylistically, since you've already shown a product result list, is it necessary to print the output again with this new list? In other words, the reader has been shown what it looks like, and now will know e.g. what columns are there. Again, it's a stylistic choice, feel free to determine for yourself what's best here.</p><p> </p><p>Then the previous comments apply below too - how did you now to choose these specific rows, etc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>